pkg/cloudprovider/providers/openstack/openstack_instances.go

```diff
@@ -215,17 +221,14 @@ func (i *Instances) InstanceType(ctx context.Context, name types.NodeName) (stri
 }
 
 func srvInstanceType(srv *servers.Server) (string, error) {
-	keys := []string{"name", "id", "original_name"}
```

Comment:
I believe you don't want to change this logic if we are going to negotiate microversion?
as we might get name,id or original_name based on the API request we send?